build/constants.js

```diff
@@ -32,6 +32,15 @@ const FOLDERSEARCH = process.env.BUILD_FOLDERSEARCH || "";
 const NO_PROGRESSBAR = Boolean(
   JSON.parse(process.env.BUILD_NO_PROGRESSBAR || process.env.CI || "false")
 );
+const FIX_FLAWS = JSON.parse(process.env.BUILD_FIX_FLAWS || "false");
+const FIX_FLAWS_DRY_RUN = JSON.parse(
+  process.env.BUILD_FIX_FLAWS_DRY_RUN || "false"
+);
```

Comment:
wondering whether we'd want to combine `FIX_FLAWS` and `FIX_FLAWS_DRY_RUN` into `FIX_FLAWS` with values of something like `("false", "dry", "write")`

Reply:
It's not a bad idea but not a great one either. It has its own set of problems. I think `FIX_FLAWS_DRY_RUN` **speaks** for itself and it's mentally easy to digest. 
I'll have to think about that for another time.

Reply:
Ok if we sit on that one for a while?